src/bacnet/basic/service/s_awfs.c

```diff
@@ -46,7 +50,12 @@ uint8_t Send_Atomic_Write_File_Stream(
         return 0;
     }
     /* is the device bound? */
-    status = address_get_by_device(device_id, &max_apdu, &dest);
+    status = address_get_by_device(
```

Comment:
this change is not needed if `address_segment_get_by_device()` is added to address.c module and address_get_by_device() remains unchanged